inputgraph/loader.go

```diff
@@ -382,7 +382,27 @@ func (l *loader) handleWait(ctx context.Context, waitStmt spec.WaitStatement) er
 }
 
 func (l *loader) handleTry(ctx context.Context, tryStmt spec.TryStatement) error {
-	return errors.New("try not supported")
+	l.hashTryStatement(tryStmt)
+	for _, stmt := range tryStmt.TryBody {
```

Comment:
I think you need to include the number of elements in each loop, otherwise how do you distinguish between

```
TRY
RUN echo hello
END
RUN echo world
```

and

```
TRY
RUN echo hello
RUN echo world
END
```

(this example assumes we'll eventually support multiple commands inside a TRY.

Reply:
Good idea. It's probably worth doing that for other statement sets.